NEWS.md

```diff
@@ -1,3 +1,30 @@
+# bbr 1.11.0
+
+## New features and changes
+
+* New bootstrap feature: `bbr` now supports the creation, management, and summary
+  of bootstrap runs. Runs are initialized and set up using `new_bootstrap_run()`  
+  and `setup_bootstrap_run()` respectively, and are tracked via the new 
+  `bbi_nmboot_model` type object. They can be submitted the same as any other 
+  model, after which they can be summarized via `summarize_bootstrap_run()`, and
+  "cleaned up" via `cleanup_bootstrap_run()`. (#671, #687, #701)
```

Comment:
comma nit-picks/suggestions: 1) also put a comma before respectively and 2) drop comma before 'and "cleaned up"'